aggregator/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,54 @@
+####
```

Comment:
Do we need so many Docker files? Also, can we use the same Dockerfile pattern in all modules?

Reply:
I will improve it in another PR :)